src/interface/src/app/services/plan-notes.service.ts

```diff
@@ -10,43 +10,77 @@ export interface Note {
   created_at: string;
 }
 
+export interface NoteModel {
+  name: string;
```

Comment:
is `name` the model name? I think this would be better as its own type
`type ModelName = 'planning_area'`